flair/embeddings.py

```diff
@@ -460,8 +462,11 @@ def embedding_length(self) -> int:
 
     def _add_embeddings_internal(self, sentences: List[Sentence]) -> List[Sentence]:
 
+        cache_path = '{}-tmp-cache.sqllite'.format(self.name) if self.cache_directory is None else os.path.join(
+            self.cache_directory, '{}-tmp-cache.sqllite'.format(os.path.basename(self.name)))
```

Comment:
This fails if self.cache_directory is not set.

Reply:
It should not as `self.cache_directory` is set to `None` by default and we are checking in this line if `self.cache_directory is None` or not.